### PR TITLE
update explanation on nr of trees in GBDT

### LINKED ISSUE #793: Correct statement on number of trees for GBDT
Exercise M6.03 states:

"Both gradient boosting and random forest models improve when increasing the number of trees in the ensemble. However, the scores reach a plateau where adding new trees just makes fitting and scoring slower."

While this statement holds true for random forests, it does not apply to gradient boosting decision trees. In GBDT, adding too many estimators can lead to overfitting. This can be demonstrated by creating a validation curve for GBDT, similar to what is done for random forests (see attached image). The result actually supports the need for hyperparameter tuning or applying early stopping, as shown at the end of exercise M6.03.
_gbdt_sklearn_mooc

### PR DESCRIPTION
**Pull request by fritshermans**
Fixes #793

I updated the text regarding the number of trees in a GBDT model.

### CODE REVIEW

**Review by ArturoAmorQ — APPROVED** [review]
Thanks again for your contribution @fritshermans. Just another couple of tweaks for a softer synonym but otherwise LGTM :)

**Comment by ArturoAmorQ** [comment]
Merging!
